app/jobs/scheduled/problem_checks.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+module Jobs
+
+  # This job runs all of the scheduled problem checks for the admin dashboard
+  # on a regular basis. To add a problem check for this scheduled job run
+  # call AdminDashboardData.add_scheduled_problem_check
+  class ProblemChecks < ::Jobs::Scheduled
+    every 10.minutes
+
+    def execute(args)
```

Comment:
```suggestion
    def execute(_args)
```

Reply:
Yeah guess not haha, I thought we had something like it too, maybe JS?